.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -165,6 +165,18 @@ steps:
     name: "validatePods"
     displayName: "Validate Pods"
 
+  - script: |
+      echo "validate pod IP assignment and check systemd-networkd restart"
+      kubectl get pod -owide -A
+      make test-validate-state
+      echo "restart CNS"
+      kubectl rollout restart ds azure-cns -n kube-system
```

Comment:
Let's add a wait as well. `kubectl rollout status ds azure-cns -n kube-system` to make sure azure-cns has restarted.